services/libfuzzer/libfuzzer.sh

```diff
@@ -14,6 +14,23 @@ cd "$WORKDIR" || exit
 # shellcheck source=recipes/linux/common.sh
 source ~/.local/bin/common.sh
 
+gcs-cat () {
+# gcs-cat bucket path
+python3 - "$1" "$2" << "EOF"
+import os
+import sys
+from google.cloud import storage
+
+client = storage.Client()
+bucket = client.bucket(sys.argv[1])
+
+blob = bucket.blob(sys.argv[2])
+print(f"Downloading gs://{sys.argv[1]}/{sys.argv[2]}", file=sys.stderr)
+with os.fdopen(sys.stdout.fileno(), "wb", closefd=False) as stdout:
```

Comment:
Why not take the destination file as an argument?

Reply:
This function is copied from the nyx image where it's used to pipe zstd data to zstdcat. I'd rather not change it given this will all be rewritten soon.